brave/brave-ios
# Fix #1683: Update verified/unverified publisher icons

## Summary of Changes

This pull request fixes issue #1683 

## Submitter Checklist:


## Test Plan:

- Enable rewards
- Visit a unverified publisher, verify round grey icon
- Visit a verified publisher (i.e. bumpsmack.com), verify rounded icon in both panel + URL bar

## Screenshots:

| Unverified | Verified |
| --- | --- |
| ![Simulator Screen Shot - iPhone 11 Pro - 2019-10-15 at 15 30 20](https://user-images.githubusercontent.com/529104/66863311-bd19e300-ef60-11e9-839b-df378277ab47.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2019-10-15 at 15 30 09](https://user-images.githubusercontent.com/529104/66863322-c0ad6a00-ef60-11e9-8559-c36ab6cfb41a.png) |

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
